spec/features/delius_import_job_feature_spec.rb

```diff
@@ -55,22 +57,32 @@
   context "when the team is associated with an LDU", :js do
     before do
       ENV['DELIUS_EMAIL_FOLDER'] = 'delius_import_feature'
-      create(:team, code: 'A', name: 'NPS - Team 1')
+      team = create(:team, code: 'A', name: 'NPS - Team 1')
+
+      stub_community_offender(offender_no, build(:community_data, offenderManagers: [build(:community_offender_manager, team: { code: team.code, localDeliveryUnit: { code: team.local_divisional_unit.code } })]))
+      stub_community_registrations(offender_no, [])
     end
 
     include_examples "imports the Delius spreadsheet and creates case information"
   end
 
-  context "when the team is not associated with an LDU" do
+  context "when the team is not associated with an LDU", :js do
     before do
       ENV['DELIUS_EMAIL_FOLDER'] = 'delius_import_feature'
```

Comment:
This env var – `ENV['DELIUS_EMAIL_FOLDER']` – is only used by `DeliusImportJob`. But it looks like this test file has now shifted focus to test `ProcessDeliusDataJob` instead. So might need some clean-up.

It's tricky because this feature spec used to test the entire import – i.e. it can read the XLSX file, update teams/LDUs, and then import offenders.

Reply:
As discussed: it makes sense to keep this feature spec until we've completely moved away from the XLSX import (and therefore have stopped the 'auto-fixing' of teams/LDUs).